Sorts/QuickSortRecursive.js

```diff
@@ -16,21 +16,36 @@
 */
 
 /**
- * Partition in place QuickSort.
- * @param {number[]} inputList list of values.
- * @param {number} low lower index for partition.
- * @param {number} high higher index for partition.
+ * Sorts the input list using the quicksort algorithm.
+ *
+ * @param {Array} inputList - the list to be sorted
+ * @return {Array} the sorted list
```

Comment:
Since we mutate the array anyways, it's unnecessary to return it, and IMO cleaner not to.

Reply:
You are absolutely right ,thanks for reminding me .